src/components/panel/panel.component.tsx

```diff
@@ -0,0 +1,84 @@
+import { Button } from "src/components/button";
+import { Expand } from "src/components/icons";
+import { Pill } from "src/components/pill";
+import { Tabs } from "src/components/tabs/tabs.component";
+import { useGetObject } from "src/hooks/useGetObject";
+import { formatObjectField } from "src/lib/utils";
+
+interface Props {
+  objectType: string;
+  closePanel: () => void;
+  uid: string;
+}
+
+export const Panel = ({ closePanel, objectType, uid }: Props) => {
+  const { data } = useGetObject(objectType, {
+    uid: uid,
+  });
+
+  const getTitle = (object: Record<string, string>, priority: string[]) => {
+    const [title] = priority.map((key) => object[key]).filter((key) => key);
+    return title;
+  };
+
+  const orderedKeys = ["title", "name", "uid"];
+
+  return (
+    <>
+      <div
+        onClick={() => closePanel()}
+        className="fixed left-0 top-0 z-50 h-full w-3/5 bg-black bg-opacity-20 "
+      />
+      <section className="fixed top-0 right-0 z-50 h-full w-full overflow-y-scroll bg-white drop-shadow-md md:w-2/3 lg:w-7/12 xl:w-2/5 ">
+        {data && (
+          <>
+            <div className="p-4 pb-2 md:p-8">
```

Comment:
The spacing, I think `p-8` is great on x axis but on `y` its too much. `md:p-8 md:py-4`? reduces the size of the header slightly which could be good for space

Reply:
left `md:py-6`. let me know if you think 4 suits better